app/models/concerns/cloneable.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Cloneable
```

Comment:
Did you check this helper - `module RenamingUtil`

Reply:
That doesn't work as specified (specification is it should work the same as experiments). Also, I think that code should be either refactored, or scrapped.